### PR TITLE
cmake: Add option to enable each adaptor by itself

### PR DESCRIPTION
**Pull request by alexmohr**
This commit adds two options to enable the udp and stdin
adaptor indendent from each other.
Therefore only the necessary component can be enabled

The program was tested solely for our own use cases, which might differ from yours.
Licensed under Mozilla Public License Version 2.0

<sup>Alexander Mohr, <alexander.m.mohr@daimler.com>, Daimler TSS GmbH, [imprint](https://github.com/Daimler/daimler-foss/blob/master/LEGAL_IMPRINT.md)</sup>

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -55,7 +55,8 @@ option(WITH_TESTSCRIPTS       "Set to ON to run CMakeLists.txt in testscripts"
 option(WITH_GPROF             "Set -pg to compile flags"                                                         OFF)
 option(WITH_DLTTEST           "Set to ON to build with modifications to test User-Daemon communication with corrupt messages" OFF)
 option(WITH_DLT_SHM_ENABLE    "EXPERIMENTAL! Set to ON to use shared memory as IPC. EXPERIMENTAL!"               OFF)
-option(WITH_DLT_ADAPTOR       "Set to ON to build src/adaptor binaries"                                          OFF)
```
Comment (ssugiura) [inline CMakeLists.txt:58]: Can you keep this cmake option so that we can keep backward compatibility?

Comment (alexmohr) [reply CMakeLists.txt:58]: yes, sure. I re-added the option. Now you can build all with `WITH_DLT_ADAPTOR` or disable single items

File: CMakeLists.txt [hunk 2]
```diff
@@ -284,7 +286,8 @@ message(STATUS "WITH_SYSTEMD_JOURNAL = ${WITH_SYSTEMD_JOURNAL}")
 message(STATUS "WITH_DOC = ${WITH_DOC}")
 
 message(STATUS "WITH_MAN = ${WITH_MAN}")
-message(STATUS "WITH_DLT_ADAPTOR = ${WITH_DLT_ADAPTOR}")
```
Comment (thanhbnq) [inline CMakeLists.txt:287]: @alexmohr 
Please also incorporate comments from @ssugiura at this place as well.
I believe that your PR is ready to go after this minor update.
Thank you

Comment (alexmohr) [reply CMakeLists.txt:287]: re-added `message(STATUS "WITH_DLT_ADAPTOR = ${WITH_DLT_ADAPTOR}")`

**Pull request merged**